pom.xml

```diff
@@ -91,14 +91,6 @@
                         </execution>
                     </executions>
                 </plugin>
-                <plugin>
-                    <groupId>org.apache.maven.plugins</groupId>
-                    <artifactId>maven-javadoc-plugin</artifactId>
```

Comment:
Why is this removed? don't we want java-doc for our rascal jars? or are we not using it anywhere anyways?

Reply:
we have never published our API online. Most of that javadoc is internal API to begin with. For vallang it could make sense but not for the Rascal project. I removed it for that reason.